packages/fetch-from-har/bin/anonymize-har.js

```diff
@@ -0,0 +1,222 @@
+const valueRegex = ( key, flags ) => {
+	return new RegExp(
+		'(?:"name": "' +
+			key +
+			'",\\s+"value": "|\b' +
+			key +
+			'=)([a-zA-Z0-9._-]{4,})',
+		flags
+	);
+};
+
+const now = dayjs();
+let firstTime;
+
+const anonymizers = {
+	timestamp: {
+		// ISO 8601
+		regex: /\b((?:[1-9]\d{3}-(?:(?:0[1-9]|1[0-2])-(?:0[1-9]|1\d|2[0-8])|(?:0[13-9]|1[0-2])-(?:29|30)|(?:0[13578]|1[02])-31)|(?:[1-9]\d(?:0[48]|[2468][048]|[13579][26])|(?:[2468][048]|[13579][26])00)-02-29)T(?:[01]\d|2[0-3]):[0-5]\d:[0-5]\d(?:[.]\d{1,5})?(?:Z|[+-][01]\d:[0-5]\d))/g,
+		replacement: ( m ) => {
+			if ( ! firstTime ) {
+				firstTime = dayjs( m );
+			}
+			// To maintain the timing between the events, we're calculating the relative timestamps here.
+			return now.subtract( firstTime.diff( m ), 'ms' ).toISOString();
+		},
+	},
+	ipv4: {
+		regex: /"((?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))"/g,
+		replacement: () => {
+			return '127.0.0.' + counter( 'ipv4' );
+		},
+	},
+	ipv6: {
+		regex: /"(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,})"/g,
+		replacement: () => {
+			return '::' + counter( 'ipv6' );
+		},
+	},
+	email: {
+		regex: /\b((?:[a-z0-9!#$%&'*+=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z](?:[a-z0-9-]*[a-z])?))/g,
+		replacement: () => {
+			return 'example' + counter( 'email' ) + '@example.com';
+		},
+	},
+	defaultUUID: {
+		regex: /(([0-9a-fA-F])\2{7}-\2{4}-\2{4}-\2{4}-\2{10}[0-9a-fA-F]{2})/g,
+		replacement: ( m ) => m,
+	},
+	uuid: {
+		regex: /\b([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})\b/g,
+		replacement: uuidv4,
+	},
+	wixSite: {
+		regex: /(?:\/|u002F)([a-z0-9]+)[.]wixsite[.]com/gi,
+		replacement: randomNumber,
+	},
+	_wixUIDX: {
+		regex: /_wixUIDX=(\d+)/g,
+		replacement: randomNumber,
+	},
+	mediaToken: {
+		regex: /__MEDIA_TOKEN__ = '([^']+)/g,
+		replacement: randomBase62String,
+	},
+	wixClient: {
+		regex: /wixClient=(\d+)/g,
+		replacement: randomNumber,
+	},
+	wixSession2: {
+		regex: /wixSession2=(JWT[.][^&;]+)/g,
+		replacement: randomBase62String,
+	},
+	siteToken: {
+		regex: valueRegex( 'site_token', 'g' ),
+		replacement: randomBase62String,
+	},
+	authorization: {
+		regex: valueRegex( 'Authorization', 'gi' ),
+		replacement: randomBase62String,
+	},
+	session_id: {
+		regex: valueRegex( 'session_id', 'g' ),
+		replacement: randomBase62String,
+	},
+	request_id: {
+		regex: valueRegex( 'x-wix-request-id', 'g' ),
+		replacement: randomBase62String,
+	},
+};
```

Comment:
There are a few other cookies (and matching headers) that we might want to anonymize or even completely remove (assuming extracting still works):

- `svSession` - "Used in connection with user login", and `XSRF-TOKEN` -"Used for security reasons" . `TS*`  - "Used for security and anti-fraud reasons" [Reference here](https://support.wix.com/en/article/cookies-and-your-wix-site) 
- `_wixAB3` - probably some ab testing
- `_ga` - Google Analytics
- `_gcl_au` - Google Tag Manager

For the headers, can we anonymize/remove:
- x-seen-by (wix specific)